domain-model/src/main/java/uk/gov/hmcts/cmc/domain/models/MediationRequest.java

```diff
@@ -0,0 +1,32 @@
+package uk.gov.hmcts.cmc.domain.models;
+
+import org.springframework.format.annotation.DateTimeFormat;
+
+import javax.validation.constraints.NotEmpty;
+import java.time.LocalDate;
+
+public class MediationRequest {
```

Comment:
Could use lombok here too.
`@AllArgsConstructor` on the class, and `@Getter` on both fields